docs/reference/generated/alert-dialog-description.json

```
@@ -3,11 +3,11 @@
   "description": "A paragraph with additional information about the alert dialog.\nRenders a `<p>` element.",
   "props": {
     "className": {
-      "type": "string | (state) => string",
+      "type": "string | ((state: {}) => string)",
```

```suggestion
      "type": "string | ((state: State) => string)",
```
It's strange that it is not consistent with the rest.

---

In this case `State` is an empty interface, that's why it's processed differently. I'll make it consistent, though.